LayoutTests/platform/gtk/TestExpectations

```diff
@@ -2657,6 +2657,8 @@ webkit.org/b/261024 webgl/2.0.y/conformance/renderbuffers/framebuffer-state-rest
 webkit.org/b/261024 webgl/2.0.y/conformance/rendering/color-mask-preserved-during-implicit-clears.html [ Pass Timeout ]
 webkit.org/b/261024 webrtc/utf8-sdp.html [ Pass Timeout ]
 
+fast/text/canceled-mousemove-does-not-prevent-selectionchange.html [ Failure ]
```

Comment:
What's up with the new failure here?

Reply:
I'm not 100% sure what happened here. It looks like _some_ type of selection happened with the defaultPrevented mousemove event, but not like on the other ports? I could file a bugzilla for further investigation.

The failure was from this EWS run by the way: https://github.com/WebKit/WebKit/pull/18890#issuecomment-1759693346

```diff
--- /home/ews/worker/GTK-WK2-Tests-EWS/build/layout-test-results/fast/text/canceled-mousemove-does-not-prevent-selectionchange-expected.txt
+++ /home/ews/worker/GTK-WK2-Tests-EWS/build/layout-test-results/fast/text/canceled-mousemove-does-not-prevent-selectionchange-actual.txt
@@ -4,5 +4,3 @@
 Pass if non-selection is visible.
 Try to select this text.
 Pass if non-selection is visible.
-Try to select this text.
-Pass if non-selection is visible.
```

Reply:
Rewrote the test and removed this failure expectation. Waiting on EWS now! 🤞